packages/page-accounts/test/pages/accountsPage.tsx

```diff
@@ -0,0 +1,135 @@
+import Overview from '../pages/../../src/Accounts/index';
+
+let queueExtrinsic: (value: PartialQueueTxExtrinsic) => void;
+
+class NotYetRendered extends Error {
+}
+
+export type AccountRow = HTMLElement;
+
+jest.mock('@polkadot/react-hooks/useAccounts', () => ({
+  useAccounts: () => mockAccountHooks.useAccounts
+}));
+
+jest.mock('@polkadot/react-hooks/useLoadingDelay', () => ({
+  useLoadingDelay: () => false
+}));
+
+jest.mock('@polkadot/react-hooks/useBalance', () => ({
+  useBalancesAll: (address: string) => mockAccountHooks.accountBalances
+}));
+
+jest.mock('@polkadot/react-hooks/useBestNumber', () => ({
+  useBestNumber: () => 1
+}));
+
+/**
+ * Account input test data
+ */
+interface ArrangedAccount {
+  id: string,
+  totalBalance: number;
+}
+
+export class AccountsPage {
+  private renderResult?: RenderResult
+
+  renderPage (accounts: ArrangedAccount[]) {
+    mockAccountHooks.setAccounts(accounts.map((account) => account.id));
+    const mockApi: ApiProps = {
+      api: {
+        derive: {
+          accounts: {
+            info: () => Promise.resolve(() => { /**/
+            })
+          },
+          balances: {
+            all: () => ({
+              accountNonce: mockAccountHooks.nonce,
+              additional: [],
+              freeBalance: 0,
+              lockedBreakdown: [],
+              reservedBalance: 0
+            })
+          },
+          chain: {
+            bestNumber: () => new BN(1) as BlockNumber
+          }
+        },
+        genesisHash: new TypeRegistry().createType('Hash', POLKADOT_GENESIS),
+        query: {},
+        registry: { chainDecimals: [12], chainTokens: ['Unit'] },
+        tx: {
+          council: {
+            // propose
+          }
+        }
+      },
+      systemName: 'substrate'
+    } as unknown as ApiProps;
+
+    queueExtrinsic = jest.fn() as QueueTxExtrinsicAdd;
+    const queue = {
+      queueExtrinsic
+    } as QueueProps;
+
+    this.renderResult = render(
+      <>
+        <div id='tooltips'/>
+        <Suspense fallback='...'>
+          <QueueProvider value={queue}>
+            <MemoryRouter>
+              <ThemeProvider theme={lightTheme}>
+                <ApiContext.Provider value={mockApi}>
+                  <Overview/>
+                </ApiContext.Provider>
+              </ThemeProvider>
+            </MemoryRouter>
+          </QueueProvider>
+        </Suspense>
+      </>
+    );
+  }
+
+  async findAccountsTable (): Promise<HTMLElement> {
+    this.assertRendered();
+    const table = await screen.findByRole('table');
+
+    return table;
+  }
+
+  async findAccountRows (): Promise<AccountRow[]> {
+    this.assertRendered();
+    const table = await this.findAccountsTable();
+    const tableBody = table.getElementsByTagName('tbody')[0];
+    const rows = await within(tableBody).findAllByRole('row');
+    const accountRows = rows.filter((r) => r.className.startsWith('Account-'));
+
+    return accountRows;
```

Comment:
Can be joined into a single return statement.

Reply:
sure